styles/main.css

```diff
@@ -24,6 +24,10 @@ h3 {
   margin-top: 2em;
 }
 
+.capitalise {
```

Comment:
I don't see `capitalise` used as a class anywhere. Was this added inadvertently?

Reply:
> I don't see `capitalise` used as a class anywhere. Was this added inadvertently?